ank/doc/swdesign/plantuml/seq_delete_workload.plantuml

```diff
@@ -23,25 +23,15 @@ end box
 
 main -> cli_commands ++ : delete workloads
 
-cli_commands ->> listen_to_manager ++: request complete state
-listen_to_manager ->> server --++ : request complete state
+cli_commands ->> listen_to_manager --++: request update state
```

Comment:
Why is there a task that listens to the Agent Manager? In the code I can only see that the CLI is sending messages to the server and receives responses from the server.

Reply:
I have checked the other CLI diagrams containing also the `listen_to_manager` and maybe this represents the fact that the CLI is treated like an agent within the GRPC communication client and is only distinguished by the communication type enum variant. But this context it is just showing a connection between CLI and Server. For a user reading the diagrams it is confusing to suddenly read about Agent Manager tasks without any further information.

Reply:
I will open a separate ticket for that so that we can merge this PR here.